{{cookiecutter.project_name }}/.circleci/config.yml

```
@@ -0,0 +1,55 @@
+version: 2.1
+
+jobs:
+  build-and-test:
+    docker:
+      - image: cimg/python:{{ cookiecutter.python_version }}
+    steps:
+      - checkout
+      - run:
+          name: Download poetry
+          command: make poetry-download
+      - run:
+          name: Install poetry
+          command: make install
```

you can use some caching layer in circleci. For instance
```suggestion
      - restore_cache:
          keys: 
            - deps-{{ checksum "poetry.lock" }}
      - run:
          name: Install poetry
          command: make install
      - save_cache:
          key: deps-{{ checksum "poetry.lock" }}
          paths: 
            - /home/circleci/.cache/pypoetry/virtualenvs
```